docs/output.md

```diff
@@ -76,6 +76,11 @@ Additionally the pipeline reports some statistics on protein and peptide numbers
 
 </details>
 
+The epitope prediction scoring system and resulting prediction scores will depend on the chosen prediction method (`--pred_method`).
+
+- [SYFPEITHI](http://www.syfpeithi.de/index.html): The score represents how well a ligand is able to bind in comparison to all annotated binders of a specific allele. Therefore, the scores are normalized to the highest scoring ligand within each allele, resulting in scores between 0 and 1. For the downstream process `prepare_entity_binding_ratios` predicted binders are called within a score of ≥0.5.
```

Comment:
Yes exactly, I can add the first score to the text as well, to make it more clear!

How would you suggest to make it more general? Would you already try to summarise the SYFPEITHI, MHCflurry and MHCnuggets and then only state the defaults and differences?

Reply:
but yes, regarding `--mhcf_mhcn_score_threshold` one could also think about generalisation, but it could also be done later ... as you wish

Reply:
Well at a later stage we could also name the threshold based on the score so instead of `--syfpeithi_score_threshold` and `--mhcf_mhcn_score_threshold` we could use `--percent_score_threshold` und `--affinity_score_threshold`. The problem here is that user may prefer one for the wrong tool, as they are connected to the `--pred_method` so we would need warnings if they are set and another method is used.

I think I would stay with method based parameter naming and documentation for now until we have more prediction methods and the parameter names need to be elongated or changed.